.github/workflows/coverage.yaml

```
@@ -3,7 +3,7 @@ name: Evmone Coverage Report
 on:
   pull_request:
     paths:
-      - "converted-ethereum-tests.txt" # This triggers the workflow only for changes in file.txt
+      - 'tests/**' # This triggers the workflow for any changes in the tests folder
```

I think we need to trigger the workflow on changes to `converted-ethereum-tests.txt`, too? Although hopefully that would be accompanied by changes in `tests/` :slightly_smiling_face: 
```suggestion
      - 'tests/**' # This triggers the workflow for any changes in the tests folder
      - "converted-ethereum-tests.txt" # This triggers the workflow only for changes in file.txt
```

---

hm, only to protect from changes in this file without coverage.